frontend/model/contracts/group.js

```diff
@@ -23,10 +23,11 @@ export const PAYMENT_TYPE_BITCOIN = 'bitcoin'
 export const PAYMENT_TYPE_PAYPAL = 'paypal'
 export const paymentType = unionOf(...[PAYMENT_TYPE_MANUAL, PAYMENT_TYPE_BITCOIN, PAYMENT_TYPE_PAYPAL].map(k => literalOf(k)))
 
-export function generateInvites (numInvites: number) {
+export function generateInvites (numInvites: number = 1, inviteeName) {
   return {
     inviteSecret: `${parseInt(Math.random() * 10000)}`, // TODO: this
-    numInvites
+    numInvites,
+    inviteeName
```

Comment:
Note: We need the inviteeName to show on links management component with real data for @SebinSong.